### PR TITLE
DM-11305: Add templates and calibs directory

### PR DESCRIPTION
**Pull request by djreiss**

### CODE REVIEW

File: calib/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Directory for (master) calibs
```
Comment (kfindeisen) [inline calib/README.md:1]: Why use a readme file instead of a dummy data file like you did with `raw/`?

**Review by kfindeisen — CHANGES_REQUESTED** [review]
I'm a bit puzzled by the readme files. Also, you forgot to add the new directories to the root readme; I don't think the template will be easy to use if the documentation doesn't match.

File: calib/README.md (hunk 1 above)
Comment (djreiss) [reply calib/README.md:1]: There are no master calibs (AFAIK) in obs_test, so not clear where to get one.
But I realized that I forgot to edit the main README so I will do that now.

**Comment by djreiss** [comment]
The readme files are simply intended as stubs to include the empty directories in the repo. I have updated the main README file.

**Review by kfindeisen — APPROVED** [review]

**Pull request merged**